sig-platform/distributed-workloads.md

```diff
@@ -0,0 +1,152 @@
+- Allows for direct submission of jobs, deployments, stateful sets, or any other Kube kind, which provides infinite customizability
+- In addition to runnable components of the app, non-runnable components such as secrets, service definitions, config maps etc. are included with the app in one holistic AppWrapper definition
+- Job meta-scheduler (job queue manager)
+- Checks for available resources, quota limits, as well as queue ordering policy (e.g., priority queue)
+- Holds the job until ready, rather than immediately creating pods and throwing responsibility onto basic Kubernetes scheduler
+- Provides the ability to implement multiple desirable quota management policies based on customer use case
+- Potential for plug-and-play/configurable resource quota management
+- Enhanced user experience for the data scientist by eliminating the need to manually coordinate with other users of a shared cluster to run a large training job that will require freeing large amounts of cluster resources
+
+*InstaScale*
+- Cluster elasticity catering for large distributed job requirements, realizing instant scaling by acquiring resources in the cloud in a demand-driven way based on queued jobs’ resource requests
+- Resources are acquired from the cloud provider at job start/finish time, and released immediately as soon as the job completes, which minimizes the cost of resources, as none are kept idle
+- Selects right provider instance sizes on behalf of the user
+
+Current state of community landscape regarding batch processing and scaling is summarized here:
+[Openshift Batch and Open Source landscape](https://docs.google.com/document/d/1N6cqIwylnDRtqHoxiq8GdwO7AbvkcNEdpWil2CzkzNU/edit?usp=sharing)
+
+**User Use Cases**
+- *Data Scientists* need self-service access to ephemeral high-performance compute infrastructure (i.e, multi-node GPU enabled clusters) for performing distributed computation. This platform supports both on-prem and cloud based environments.
+- *Cluster Administrators* can provide the capacity for on-demand infrastructure to their data scientist users, while implementing per user quota policies for fair access and limiting cloud spend. 
+- *Cluster Users’* requests of infrastructure for distributed compute jobs or interactive development environments will be placed in a queue until the necessary cluster resources become available.      
+
+## Architecture
+The architecture and typical workflow for a sample Ray distributed workload are depicted below.
+![Screenshot from 2022-12-02 14-57-05](https://user-images.githubusercontent.com/43946617/206355518-4b3ecd06-7290-4c4c-9f21-8bfb02011927.png)
+
+------------------------------------------------------------------------------------------------------------------------------
+
+![Screenshot from 2022-12-02 14-56-13](https://user-images.githubusercontent.com/43946617/206355725-96eba2dd-1b2f-489e-a396-981bdc6f0819.png)
+
+## Implementation
```

Comment:
@heyselbi Is this community document going to be the single source of truth for this integration component, if not then this implementation should be stored with the required deployment files for the integration

Reply:
Removed most of the implementation and left the demo part. I also added the readme file to wg-distributed-workloads directory